src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -126,6 +132,16 @@ const mapOfferToOption = (offer: ExchangeOffer) => {
   };
 };
 
+const mapRouteToOption = (route: Route) => {
+  const [fistStep] = route.steps;
+  const serviceDetails = bridgeServiceIdToDetails[fistStep?.toolDetails?.key ?? 'lifi'];
+  return {
+    title: fistStep?.toolDetails?.name ?? serviceDetails?.title ?? 'LiFi',
+    value: route.id,
+    iconUrl: fistStep?.toolDetails?.logoURI ?? serviceDetails?.iconUrl,
```

Comment:
So this condition here, tell me if I'm mistaken, since the first part can be undefined ie `fistStep?.toolDetails?.logoURI` it also seems the second part can be undefined too ie `serviceDetails?.iconUrl` so are we sure that if first part doesn't exist then the second part will definitely exist?

Reply:
@MayukhSamadder2000  serviceDetails is derived from an object constant **bridgeServiceIdToDetails** like this:
`const serviceDetails = bridgeServiceIdToDetails[fistStep?.toolDetails?.key ?? 'lifi'];
`
So serviceDetails will always be there taking up 'lifi' as the default key if the first value is undefined.